tests/libcxx/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+set(LIBCXX_SOURCE_DIR "${LLVM_PROJECT_SOURCE_DIR}/libcxx" CACHE PATH

```

Comment:
@barcharcraz You should create a separate comment on the same line for a separate issue, instead of a reply, so they can be discussed and resolved independently.